lib/fastlane/plugin/appcenter/actions/appcenter_upload_action.rb

```diff
@@ -325,8 +325,6 @@ def self.run_dsym_upload(params)
             UI.message("dSYM files zipped")
           end
 
-          values[:dsym_path] = dsym_path
```

Comment:
This part is used only in tests, so I'm not sure why it caused a crash for you.

Instead of calling Action method `run_dsym_upload` consider using following:
```ruby
appcenter_upload({
  api_token: 'xxx',
  owner_name: 'owner',
  app_name: 'app',
  upload_dsym_only: true,
  dsym: './spec/fixtures/symbols/Themoji.dSYM.zip'
})
```